back_end/src/models/class_schedule.py

```diff
@@ -0,0 +1,107 @@
+from __future__ import annotations
+
+from typing import List
+
+from datetime import time
+
+from ..setup import db
+
+
+class ClassSchedule(db.Model):
+    __tablename__ = "ClasSchedule"
+
+    id = db.Column(db.Integer, primary_key=True)
+    class_id = db.Column(db.Integer, unique=True, nullable=False)

```

Comment:
I thought this would be a foreign key? To `Class.id`. So maybe `db.Column(db.Integer, unique=True, ForeignKey="Class.id", nullable=False)`